src/components/SideBar/index.js

```diff
@@ -17,24 +18,40 @@ const SideBar = () => (
           <SideBarWrapper>
             <ul>
               <li>
-                <img src={avatar} alt="avatar profile"/>
+                <img src={avatar} alt="avatar profile" />
                 <Link to="/profile/edit">Edit Profile</Link>
               </li>
-              <li><Link to="/reports/new">Create Daily Report</Link></li>
+              <li>
+                <Link to="/reports/new">Create Daily Report</Link>
+              </li>
+              <li>
+                <Link to="/reports">Your Daily Reports</Link>
+              </li>
 
               {roles.includes('TEAM_LEADER') && (
                 <Fragment>
-                  <li><Link to="/weekly-reports/new">Create Weekly Report</Link></li>
-                  <li><Link to="/statistic">Statistic</Link></li>
+                  <li>
+                    <Link to="/weekly-reports/new">Create Weekly Report</Link>
+                  </li>
+
+                  {!isEmpty(team) && (
```

Comment:
I think u also need check `team` to avoid `undefined` like this: `{team && !isEmpty(team) && (` ?

Reply:
I'm not sure about this because destructing values from GraphQL query result always returns `null` if the field value is not set.

In our application, an initialized user always have a `team` field with `null` value and that means it always we never get undefined.

But yeah, for some reason `undefined` could happen and screw us. And it's not a bad thing to add a more secure condition to this section. I will fix here with your suggestion `{team && !isEmpty(team) && ( ?` .

Reply:
FIxed